### PR TITLE
MAINT: add bug and science issue templates

### PR DESCRIPTION
**Pull request by dennissergeev**
Two templates for now.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+to gain insight at a later date - your contributions here are significant
+to the community and much appreciated!
+
+Please always remember to be courteous to others posting on this forum as
+we are all here voluntarily to help others use Isca.
+
+NOTE:
+Isca team members are all full-time research scientists, so while we are very happy to provide
+some support, response times may vary. If you need in-depth support, then we may suggest that one
+ of us becomes a co-author on your next paper by providing a long-term support on the issue.
```
Comment (sit23) [inline .github/ISSUE_TEMPLATE/bug_report.md:21]: I'd maybe change `providing a long-term support on the issue` to just `providing long-term support on the issue`.

File: .github/ISSUE_TEMPLATE/science_issue.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+to gain insight at a later date - your contributions here are significant
+to the community and much appreciated!
+
+Please always remember to be courteous to others posting on this forum as
+we are all here voluntarily to help others use Isca.
+
+NOTE:
+Isca team members are all full-time research scientists, so while we are very happy to provide
+some support, response times may vary. If you need in-depth support, then we may suggest that one
+ of us becomes a co-author on your next paper by providing a long-term support on the issue.
```
Comment (sit23) [inline .github/ISSUE_TEMPLATE/science_issue.md:21]: Same as comment on other template

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+-->
+
+-----
+**Screenshots**
+<!--
+If applicable, add screenshots to help explain your problem.
+-->
+
+-----
+**System Information:**
```
Comment (sit23) [inline .github/ISSUE_TEMPLATE/bug_report.md:41]: Perhaps we could also ask about their most recent commit ID / are they working off their own fork or just using the ExeClim one?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (dennissergeev) [reply .github/ISSUE_TEMPLATE/bug_report.md:21]: those pesky articles as always! thanks)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (dennissergeev) [reply .github/ISSUE_TEMPLATE/bug_report.md:41]: yes, good point!

**Pull request merged**